bootstrap/api/transport.go

```diff
@@ -105,7 +105,7 @@ func MakeHandler(svc bootstrap.Service, reader bootstrap.ConfigReader) http.Hand
 }
 
 func decodeAddRequest(_ context.Context, r *http.Request) (interface{}, error) {
-	if r.Header.Get("Content-Type") != contentType {
+	if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
```

Comment:
The same is applicable to other places in `transport.go` where `r.Header.Get("Content-Type") != contentType` occurs.